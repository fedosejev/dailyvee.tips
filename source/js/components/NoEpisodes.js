import React from 'react';

const NoEpisodes = () => (
  <div className="noEpisodes">No episodes found.</div>
);

export default NoEpisodes;
