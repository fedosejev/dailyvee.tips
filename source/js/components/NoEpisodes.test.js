import React from 'react';
import NoEpisodes from './NoEpisodes';
import renderer from 'react-test-renderer';

describe('NoEpisodes', () => {
  test('SHOULD render correctly', () => {
    const tree = renderer.create(
      <NoEpisodes />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
