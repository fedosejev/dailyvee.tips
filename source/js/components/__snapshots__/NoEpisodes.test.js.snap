// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoEpisodes SHOULD render correctly 1`] = `
<div
  className="noEpisodes"
>
  No episodes found.
</div>
`;
